src/pages/createProfile/createProfileSteps/SelectRole.tsx

```diff
@@ -1,16 +1,19 @@
 import { FunctionComponent } from "react";
 import { AddProfileInformation } from "components";
 import { useSelector } from "react-redux";
+import { FormikValues, useFormikContext } from "formik";
 import { selectContent } from "modules/content";
 
 const SelectRole: FunctionComponent = () => {
   const { roles } = useSelector(selectContent);
+  const { values } = useFormikContext();
 
   return (
     <AddProfileInformation
       fieldName="role"
-      prompt="What is your main role?"
       helperText="Type or select a role"
+      placeholder="role"
+      prompt={ `What is your main role, ${(values as FormikValues).nickname}?` }
```

Comment:
found that the mock had the nickname in there so i added it here, let me know if i need to revert this change